lib/mods/origin_info.rb

```diff
@@ -13,7 +13,9 @@ def dates
     end
 
     def key_dates
```

Comment:
Please add comments about what this returns.  Presently it looks like it's dates marked `keyDate` and ranges?  Is this still the right method name?

Reply:
Yes, this is still key dates. The MODS spec says this is how the `keyDate` flag works.

Reply:
I guess the problem I'm seeing here is that it appears to select _any_ range, not just one tagged `keyDate`. Is that right?

Reply:
Oh, I see how it's working now.